docs/DEPLOY.md

```diff
@@ -257,92 +257,206 @@ There is an example nginx configuration provided at `config/deploy/nginx.conf` a
+```
+
+#### 4. Podman
+
+Add the `rootless-podman-service-type` and it DBus dependencies to your configuration in the `services` field:
+
+```scheme
+(use-modules (gnu services containers) ;for 'rootless-podman-service-type'
+             (gnu services dbus)       ;for 'dbus-service-type'
+             (gnu services desktop)    ;for 'elogind-service-type'
+             ...)
+
+...
+
+;; The DBus clique
+(service elogind-service-type)
+(service dbus-root-service-type)
+
+;; Podman
+(service rootless-podman-service-type)
+
+;; This service implements OCI provisioned Shepherd services
+(service oci-service-type
+         (oci-configuration
+          (runtime 'podman)))
 ```
 
+#### 5. Meilisearch and Bonfire
+
+To provision a functional Bonfire instance you will need two services, meilisearch and the Bonfire flavour. The `(bonfire services bonfire)` module provides a Guix System service able to provision a database and the Bonfire instance. Add the following to your `operating-system` configuration, and make sure to change the values based on your actual setup:
+
+```scheme
+(use-modules (bonfire services bonfire) ;for 'oci-bonfire-service-type'
+             (oci services meilisearch) ;for 'oci-meilisearch-service-type'
+             ...)
+
+...
+
+;; meilisearch
+(service oci-meilisearch-service-type
+         (oci-meilisearch-configuration
+          (network "host")
+          (master-key "/path/to/meilisearch/master/key/secret")))
+
+;; Bonfire
+(service oci-bonfire-service-type
+         (oci-bonfire-configuration
+          (configuration
+           (bonfire-configuration
+            ;; Networking
+            (hostname "yourdomain.net")  ;replace with your domain
+            (public-port "443")
+            (network "host")
+            ;; Postgres
+            (postgres-user "bonfire")
+            (postgres-db "bonfire")
+            ;; Mail
+            (mail-domain "yourdomain.net")
+            (mail-from "youremail@address.org")))
+          ;; Shepherd dependencies
+          (requirement
+           '(user-processes postgresql postgres-roles podman-meilisearch))
+          (extra-variables
+           `(("MAIL_BACKEND" . "mailjet")
+             ("SERVER_PORT" . "4000")
+             ("SEARCH_MEILI_INSTANCE" . "http://localhost:7700")))
+          ;; Secrets
+          (meili-master-key
+           "/path/to/meilisearch/master/key/secret")
+          (postgres-password
+           "/path/to/postgres/password/secret")
+          (mail-key
+           "/path/to/mail/key/secret")
+          (mail-private-key
+           "/path/to/mail/private/key/secret")
+          (secret-key-base
+           "/path/to/secret/key/base/secret")
+          (signing-salt
+           "/path/to/signing/salt/secret")
+          (encryption-salt
+           "/path/to/encryption/salt/secret")))
+```
+
+One option to provision secrets as files in a Guix aware way is [sops-guix](https://github.com/fishinthecalculator/sops-guix).
```

Comment:
maybe moving that to the beginning of this section with an example of how to set up a secret using it?

Reply:
Yeah recently got feedback on our coopcloud guide that there's too much jumping between different guides/tabs when following it, so I'd be for copying the steps here.

Reply:
I added the steps to setup secrets, let me know if it's clear/it's too much material. thank you!